docarray/array/storage/weaviate/getsetdel.py

```diff
@@ -16,11 +16,19 @@ def _getitem(self, wid: str) -> 'Document':
         :raises KeyError: raise error when weaviate id does not exist in storage
         :return: Document
         """
-        try:
-            resp = self._client.data_object.get_by_id(wid, with_vector=True)
-            return Document.from_base64(
-                resp['properties']['_serialized'], **self._serialize_config
-            )
+        try:   
+            resp = self._client.data_object.get_by_id(wid, class_name=self._config.name ,with_vector=True)
+
+            return_doc = Document(id=resp['properties']['_docarray_id'])
+
+            for prop in resp['properties']:            
+                if prop[:1] != '_':
+                    setattr(return_doc, prop, resp['properties'][prop])
+
+            if 'vector' in resp and resp['vector'] != None:
```

Comment:
add a similar syntax as the one proposed before

Reply:
Sorry, not 100% what you refer to here 🙈

Reply:
I mean, use better the dict.get syntax proposed in another change